metricbeat/module/kubernetes/state_cronjob/_meta/testdata/ksm.v2.4.2.plain

```diff
@@ -0,0 +1,1422 @@
+# HELP kube_certificatesigningrequest_annotations Kubernetes annotations converted to Prometheus labels.
```

Comment:
We can't, because the test that uses those fields looks for the folder `testdata` and the file extension .plain. It is a bad solution because we are copying many of the same files.

Reply:
understood, thank you for explanation!